Makefile.base

```diff
@@ -57,10 +57,12 @@ CXXFLAGS = $(filter-out $(CXXUWFLAGS), $(CFLAGS)) $(CXXEXFLAGS)
 # compile and generate dependency info
 
 $(OBJC): $(BINDIR)$(MODULE)/%.o: %.c
-	$(AD)$(CC) $(CFLAGS) $(INCLUDES) -MD -MP -c -o $@ $(abspath $<)
+	$(AD)$(CC) -D__SHORT_FILE__=\"$(subst $(RIOTBASE)/,,$(abspath $<))\" \
```

Comment:
I think that a more robust way would be to do `$(patsubst $(RIOTBASE)%/,,$(abspath $<))`